ClassificationNeuralNetwork.java

```diff
@@ -62,4 +82,41 @@ public static void readDataFromFile()
 					+ currLineNum);
 		}
 	}
+	
+	public static void putDataInArrayList()
+	{
+		for(int i=0; i<numLines; i++)
```

Comment:
To make this method easy to reuse, I would put lines 34-38 in this method instead of in the main

Reply:
Changed. (I'm doing this so I can keep track of what I have and haven't changed)